client/javascript/scrabble.js

```diff
@@ -557,13 +848,26 @@ function LetterBag()
 {
 }
 
-LetterBag.create = function(language) {
+LetterBag.create = function(language, size) {
     var letterBag = new LetterBag;
+    var langPlus = language;
+    switch (size) {
+		    	case 'Regular':
+                                boardLen=15;
+                                break;
+	    		default:
+                                boardLen=21;
+		    		langPlus += "-SS";
+                                break;
+    }
```

Comment:
Again, please make "Regular" be the default or have two explicit cases.  Also indentation.

Reply:
done